scalableminds/webknossos
# Use correct compare method for numbers

I didn't manage to change the flowtypes for the compareBy methods, so that one can only be called with numbers and the other with strings. However, I fixed our usage of compareBy and localeCompareBy in the admin views and added a console error message to warn of wrong usage in the future.

### URL of deployed dev instance (used for testing):
- https://repairsorting.webknossos.xyz

### Steps to test:
- Open the admin views that are affected by this PR and check whether the sorting works correctly.

### Issues:
- fixes incorrect sorting in some admin views

------
